src/recipients/RecipientPicker.tsx

```diff
@@ -92,6 +92,7 @@ function RecipientPicker(props: RecipientProps) {
 
   const isFetchingVerificationStatus = (recipient: Recipient) => {
     return (
+      props.selectedRecipient &&
       recipient.e164PhoneNumber === props.selectedRecipient?.e164PhoneNumber &&
```

Comment:
this line caused the loader to appear - for contacts with address only, the `e164PhoneNumber` is undefined. same as the `selectedRecipient.e164PhoneNumber` since `selectedRecipient` is undefined when the screen first loads and the user hasn't selected a contact to send to